server/src/storage/customers.ts

```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  */
 
-import {Datastore} from '@google-cloud/datastore';
-import {Guid} from 'guid-typescript';
+import {CUSTOMER_KIND} from '../constants/kinds';
+import {CustomerResponse} from '../interfaces';
+import {getWithId} from './datastore';
 
-const datastore = new Datastore();
+const getCustomer = async (customerId: string): Promise<CustomerResponse> => {
+  return getWithId(CUSTOMER_KIND, customerId);
```

Comment:
You already have =>, do you still need {return;} block?

Reply:
Oh yea I will remove that!